projects/plugins/protect/src/js/components/textarea/index.jsx

```diff
@@ -0,0 +1,33 @@
+import styles from './styles.module.scss';
+
+const Textarea = ( {
```

Comment:
Nit-pick - `TextArea`?

Reply:
Intentionally one word to match the `<textarea />` HTML tag: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/textarea :+1: